src/url.js

```diff
@@ -67,7 +67,7 @@ export function parseUrl(url) {
  * @param {string} paramString
  * @return {string}
  */
-function appendParamStringToUrl(url, paramString) {
+export function appendParamStringToUrl(url, paramString) {
```

Comment:
Doing this is risky and can introduce XSS issues. For example if the paramString had a `"><script>js code</script>` it could execute arbitrary code on the page. 

This PR does not introduce the risks though. I construct the paramString safely and then append it in the right place. 

I see two ways forward: 
1. We make the change but add documentation to make note of this. 
2. I copy the function into amp-analytics. That way, the exposure is contained and someone can't inadvertently introduce XSS risk.

A third solution that I don't want to do is to deconstruct the paramString and then reconstruct it.

@cramforce Do you know anyone I can involve to check for XSS risks?